### PR TITLE
link missmatch  at students/sunday-class.md

### PR DESCRIPTION
**Pull request by aame1**
`line 33` points to a 404 page.
replaced it with the from [asking-for-help.md](./asking-for-help.md). :)

### CODE REVIEW

File: students/sunday-class.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ This is a co-learning moment for each one of you.
 
 Your coaches will spend some time leading a review session of last week's assignments.
 
-The best way for you to help your coaches give a helpful review session is to **create [Sunday Review Issues](https://github.com/aame1/home/blob/master/students/asking-for-help.md)** in your class repo.  This way they can prepare clear examples and presentations ahead of time.
+The best way for you to help your coaches give a helpful review session is to **create [Sunday Review Issues](https://github.com/HackYourFutureBelgium/home/blob/master/students/asking-for-help.md)** in your class repo.  This way they can prepare clear examples and presentations ahead of time.
```
Comment (colevandersWands) [inline students/sunday-class.md]: Good catch!  can you update your PR one more time?  Right now you have an absolute link to the source file which will open in a new tab.  If you change it to a relative link (`./asking-for-help.md`) then the link will stay in the tab and open a new page in the gitbook.

this mistake you found is from before the gitbook, when all the guides were in separate repos

**Comment by aame1** [comment]
hi @colevandersWands 

committed the new relative link, but, i can only "view pull request" now. :) 
I'm still not very familiar with conflicts and these scenarios. :hatching_chick:

File: students/sunday-class.md (hunk 1 above)
Comment (aame1) [reply students/sunday-class.md]: hi @colevandersWands 

committed the new relative link, but, i can only "view pull request" now. :) 
I'm still not very familiar with conflicts and these scenarios. :hatching_chick:

**Comment by colevandersWands** [comment]
@aame1 , we'll be practicing this in class the coming weeks.  its tricky stuff so no worries if you're still a bit confused.  

Having managed this PR you're already doing great

**Comment by colevandersWands** [comment]
the page has changed since this PR, closing as outdated

**Pull request closed without merging**